modules/oe_translation_cdt/src/Model/Transaction.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_translation_cdt\Model;
+
+use Symfony\Component\Serializer\Attribute\SerializedPath;
+
+/**
+ * Defines the DTO class for transaction.
+ */
+class Transaction {
+
+  #[SerializedPath('[TransactionHeader][SenderDetails][@drupalVersion]')]
+  /**
+   * The version of Drupal core.
+   */
+  protected string $drupalVersion;
+
+  #[SerializedPath('[TransactionHeader][SenderDetails][@moduleVersion]')]
+  /**
+   * The version of the module.
+   */
+  protected string $moduleVersion;
+
+  #[SerializedPath('[TransactionHeader][SenderDetails][ProducerDateTime]')]
+  /**
+   * The date of the file.
+   *
+   * @var \DateTimeInterface
+   */
+  protected \DateTimeInterface $producerDateTime;
+
+  #[SerializedPath('[TransactionIdentifier]')]
```

Comment:
I could possibly create a `NameConverter` class here to support the automatic conversion from `camelCase` to `PascalCase`, but the question is if it makes sense for few properties. Redefining the `SerializePath` works identically.